pom.xml

```diff
@@ -113,8 +130,15 @@
             <dependency>
                 <groupId>org.assertj</groupId>
                 <artifactId>assertj-core</artifactId>
+                <scope>test</scope>
                 <version>${version.org.assertj}</version>
             </dependency>
+            <dependency>
+                <groupId>org.pgpainless</groupId>
```

Comment:
is that a new dependency in the WildFly ecosystem? (it's the first I hear from it)

Reply:
Yes it's a new dependency. I used it for writing tests only as it makes GPG key operations much easier. If that's a problem I can re-write the tests to remove it